docs/building-blocks/extending-templates/serial-copy-loop.md

```diff
@@ -0,0 +1,150 @@
+
+The reason we need an "anchor" resource becomes apparent in the output of the second resource. The output value references the previous resource in the copy loop by building up the resource name using the name prefix `loop-` and the result of the `copyIndex()` function. However, there is no resource named `loop-0` because, as you recall, we started naming at `copyIndex(1)`. Because we have an "anchor" resource, on the first iteration the serial copy loop can refer to it's name and the template will pass validation. 
+
+```json
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "numberToDeploy": {
+      "type": "int",
+      "minValue": 2,
+      "defaultValue": 5
+    }
+  },
+  "variables": {
+    "count": "[sub(parameters('numberToDeploy'), 1)]"
+  },
+  "resources": [
+    {
+      "apiVersion": "2015-01-01",
+      "type": "Microsoft.Resources/deployments",
+      "name": "loop-0",
+      "properties": {
+        "mode": "Incremental",
+        "parameters": {},
+        "template": {
+          "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+          "contentVersion": "1.0.0.0",
+          "parameters": {},
+          "variables": {},
+          "resources": [],
+          "outputs": {
+            "collection": {
+              "type": "string",
+              "value": "loop-0 "
+            }
+          }
+        }
+      }
+    },
+    {
+      "apiVersion": "2015-01-01",
+      "type": "Microsoft.Resources/deployments",
+      "name": "[concat('loop-', copyIndex(1))]",
+      "dependsOn": [
+        "loop-0"
+      ],
+      "copy": {
+        "name": "iterator",
+        "count": "[variables('count')]",
+        "mode": "serial"
+      },
+      "properties": {
+        "mode": "Incremental",
+        "template": {
+          "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+          "contentVersion": "1.0.0.0",
+          "parameters": {},
+          "variables": {},
+          "resources": [],
+          "outputs": {
+            "collection": {
+              "type": "string",
+              "value": "[concat(reference(concat('loop-',copyIndex())).outputs.collection.value,'loop-',copyIndex(1), ' ')]"
+            }
+          }
+        }
+      }
+    }
+  ],
+  "outputs": {
+    "result": {
+      "type": "string",
+      "value": "[reference(concat('loop-',variables('count'))).outputs.collection.value]"
+    }
+  }
+}
+
+```
+## Try the template
+
+If you would like to experiment with this template, follow these steps:
```

Comment:
I would probably just say 'To experiment with this template...'. Maybe I am being too fussy.